.github/codeql.yaml

```diff
@@ -1,6 +1,7 @@
 paths-ignore:
   - src/pkg/packager/network.go
   - src/pkg/utils/network.go
+  - src/pkg/utils/credentials.go
```

Comment:
There is also a call that prints the push password in [src/cmd/tools.go](https://github.com/defenseunicorns/zarf/pull/1220/files#diff-69586b09df01b25b91a3d2d98bdec15c9104bbe387f369c4a46aae62fa027b03).  I would recommend making it so that this calls a credentials.go function to print this so both can be ignored.

Reply:
updated with [484cc98](https://github.com/defenseunicorns/zarf/pull/1220/commits/484cc980a35e6c9f496ff5239b3ac05a3054f98f)